packages/convex-component/example/src/App.css

```
@@ -0,0 +1,685 @@
+.chip-warn { background: var(--warn-dim); color: var(--warn); }
+.chip-danger { background: var(--danger-dim); color: var(--danger); }
+.chip-neutral { background: var(--neutral-dim); color: var(--neutral); }
+.chip-pending { background: var(--pending-dim); color: var(--pending); }
+.chip-brand { background: var(--primary-dim); color: var(--primary); }
+
+.chip-pending::before {
+  content: "";
+  width: 6px;
+  height: 6px;
+  border-radius: 50%;
+  background: currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)